utils/build/docker/nodejs/express4.base.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:18-alpine
+
+RUN apk add --no-cache bash curl git jq
+
+WORKDIR /usr/app
+
+COPY utils/build/docker/nodejs/install_ddtrace.sh binaries* /binaries/
```

Comment:
Do not ship anything from system-tests source in base images, it must be added in the build step.

The only thing that can be done is apt-get/apk & co